### PR TITLE
loottracker: Add fishing level to slayer chest metadata

### PR DESCRIPTION
**Pull request by cdfisher**
Add player's fishing level to metadata submitted for Larran's chest and brimstone chest loot events for OSRS Wiki crowdsourcing efforts. Originally mentioned in the Runescape Wikis Discord.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 1]
```diff
@@ -176,6 +176,13 @@ public class LootTrackerPlugin extends Plugin
 		put(13117, "Rogues' Chest").
 		build();
 
+	// Chests opened with keys from slayer tasks
+	private static final Set<String> SLAYER_CHEST_EVENT_TYPES = new ImmutableSet.Builder<String>().
+		add("Brimstone Chest").
+		add("Larran's big chest").
+		add("Larran's small chest").
+		build();
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:184]:
```suggestion
	private static final Set<String> SLAYER_CHEST_EVENT_TYPES = ImmutableSet.of(
		"Brimstone Chest",
		"Larran's big chest",
		"Larran's small chest"
	);
```

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 2]
```diff
@@ -710,17 +717,17 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
...
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_5_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_6_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_7_HEALTH), 1);
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_1_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_2_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_3_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_4_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_5_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_6_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_7_HEALTH), 1);
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:726]: Revert the indentation change here

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 3]
```diff
@@ -710,17 +717,17 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
...
 				int raidDamage = client.getVarbitValue(Varbits.TOA_RAID_DAMAGE);
 				event = TOMBS_OF_AMASCUT;
 				container = client.getItemContainer(InventoryID.TOA_REWARD_CHEST);
-				metadata = new int[]{ raidLevel, teamSize, raidDamage };
+				metadata = new int[]{raidLevel, teamSize, raidDamage};
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:730]: Revert this whitespace change as well

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java (hunk 2 above)
Comment (cdfisher) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: Not sure how this and the other whitespace change happened, reverted in b8175f742a5d8c62bd439918800fcf0b8a97015b

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java (hunk 3 above)
Comment (cdfisher) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: Reverted in b8175f742a5d8c62bd439918800fcf0b8a97015b

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java (hunk 1 above)
Comment (cdfisher) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: Changed in b8175f742a5d8c62bd439918800fcf0b8a97015b, thanks.

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java (hunk 3 above)
Comment (raiyni) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java]: You don't need to comment when you make these changes. We get notifications and can see the commit diffs.

**Review by raiyni — APPROVED** [review]

**Pull request merged**